src/lmnt/api.py

```diff
@@ -16,12 +16,15 @@
 import aiohttp
 import json
 
+from loguru import logger
```

Comment:
I don't see the logger used in this change, so remove?

Reply:
Done. May add it back in future PR once there's logging that feels worth leaving in (I had some, but removed it).